nextcloud-ios-CalDAVfix (#559)
Start9Labs/documentation: site/source/service-guides/nextcloud/nextcloud-setup/nextcloud-ios.rst

3. Enter the following fields and tap "Next":

>   - Server - You can find complete path in Nextcloud -> Calendar settings -> Copy iOS/macOS CalDav address. For setting up contacts/CardDav use the same path.

Reviewer: ```suggestion
  - Server - Copy your URL from Nextcloud -> Calendar settings -> Copy iOS/macOS CalDav address. For setting up contacts/CardDav use the same path.
```
